jgrapht-opt/src/main/java/org/jgrapht/opt/graph/fastutil/FastutilMapGraph.java

```diff
@@ -7,11 +7,24 @@
 
 /**
  * A graph implementation using fastutil's map implementations for storage.
+ * 
+ * <p>The following example creates a simple undirected weighted graph: <blockquote>
+ * 
+ * <pre>
+ * Graph&lt;Integer,
+ *     DefaultWeightedEdge&gt; g = new FastutilMapGraph&lt;&gt;(
+ *         SupplierUtil.createIntegerSupplier(), SupplierUtil.createDefaultWeightedEdgeSupplier(),
```

Comment:
Better to use createStringSupplier for this example since for int we steer them to use FastutilMapIntVertexGraph below.